bot.go

```diff
@@ -1252,6 +1252,26 @@ func (b *Bot) MyShortDescription(language string) (*BotInfo, error) {
 	return b.botInfo(language, "getMyShortDescription")
 }
 
+func (b *Bot) GetStarTransactions(offset, limit int) (*StarTransaction, error) {
```

Comment:
Method returns an array of Star Transactions. Rewrite `resp` struct to return an array of transactions
```go
var resp struct {
    Result struct {
       Transactions []StarTransaction  `json:"transactions"`
  }
}
```